subsys/bluetooth/Kconfig

```diff
@@ -106,6 +106,16 @@ config BT_BROADCASTER
 
 endmenu
 
+config BT_LIM_ADV_TIMEOUT
+	int "Timeout for limited advertising in 1s units"
+	default 30
+	range 1 60
```

Comment:
The GAP limit is 180s, I guess you went with 60 as per recommendation, but I don't think we need to disallow applications to set it to the maximum allowed value. 
 
```suggestion
	range 1 180
```
![image](https://user-images.githubusercontent.com/2520413/124746439-26a20800-df21-11eb-9da3-15c57d7a18f1.png)

Reply:
Could there be a valid argument for no timeout to be applied to advertising? If the application required the device to connect before initiating main functionality would it not be a good idea to constantly transmit?

Would it be possible to allow a zero (0) option which would not enforce a timeout period?

Reply:
This timeout is used only for limited advertising, which by definition should be limited by maximum advertising time. There is still possibility to advertise infinitely: just don't set limited advertising flag.